src/translations/strings.yaml

```diff
@@ -151,7 +151,9 @@ splittunnel:
   infoCardDescription2:
     value: VPN must be off to edit Excluded apps
     comment: Displays inside the information card at the top of the Excluded apps screen when the VPN is on
-
+  btnShowSystemApp: 
+    value: Show System apps
```

Comment:
Is there as reason for System being uppercase?

Reply:
Nope! Adjusted that :)